rascal-vscode-extension/src/test/vscode-suite/dsl.test.ts

```diff
@@ -146,5 +147,21 @@ describe('DSL', function () {
         await lens!.click();
         await driver.wait(async () => (await editor.getTextAtLine(9)).trim() === "b := 2;", 20_000, "a variable should be changed to b");
     });
+
+    it("quick fix works"), async() => {
```

Comment:
ah, there is an error, the `)` should not be there, this disabled the test from running it should be `it("quick fix works", ...`

Reply:
Thanks! Good catch